stacks/api/bucket/bucket.schema.resolver.spec.ts

```diff
@@ -69,35 +70,29 @@ describe("bucket service", () => {
     schemaResolver = new BucketSchemaResolver(bs);
   });
 
-  afterAll(async () => {
-    await module.close();
-  });
+  afterAll(async () => await module.close());
 
-  it("should resolve bucket", async () => {
-    const response = await schemaResolver.resolve(bucket._id.toHexString());
-    expect(response).toEqual({
+  it("should return the compiled bucket schema", async () => {
+    const jsonSchema = await schemaResolver.resolve(bucket._id.toHexString());
+    console.log(jsonSchema);
```

Comment:
Probably you forgot to delete this console.log

Reply:
Yes. but it's okay. No need to remove it because this is a test file.